R/reexport.R

```diff
@@ -5,6 +5,10 @@ dplyr::`%>%`
 
 
 # gtsummary ------------------------------------------------------------------------
+#' @export
```

Comment:
should this be `# @reexport`?

Reply:
Oh I've never used that tag before! 🤯  I did a quick search and didn't see documentation for it. Can you point me to an example?

Reply:
yep you are right just doing the above makes them a reexport! I got confused with the automatic reference title:

https://www.danieldsjoberg.com/gtsummary/reference/reexports.html